lib/sqlalchemy/orm/session.py

```diff
@@ -3580,6 +3580,129 @@ def get(
             bind_arguments=bind_arguments,
         )
 
+    def get_one(
+        self,
+        entity: _EntityBindKey[_O],
+        ident: _PKIdentityArgument,
+        *,
+        options: Optional[Sequence[ORMOption]] = None,
+        populate_existing: bool = False,
+        with_for_update: ForUpdateParameter = None,
+        identity_token: Optional[Any] = None,
+        execution_options: OrmExecuteOptionsParameter = util.EMPTY_DICT,
+        bind_arguments: Optional[_BindArguments] = None,
+    ) -> _O:
```

Comment:
**mike bayer** (zzzeek) wrote:

why do we need kw here when get() does not have them ?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4872

Reply:
Added mainly to avoid following changes to the poxied code in case additional arguments are added. If the idea is "public method should not have **kw args" then we can remove them

Reply:
**mike bayer** (zzzeek) wrote:

the proxied methods are automatically generated, arent they ?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4872

Reply:
**Federico Caselli** (CaselIT) wrote:

yes, but asyncio/session.py is not a proxy or at least ti has no generated method

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4872

Reply:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4872